pt-br/tutorials/tour/generic-classes.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: tutorial
+title: Classes Genéricas
+
+disqus: true
+
+tutorial: scala-tour
+num: 17
+tutorial-next: variances
+tutorial-previous: sequence-comprehensions
+language: pt-br
+---
+
+Semelhante ao Java 5 (aka. [JDK 1.5](http://java.sun.com/j2se/1.5/)), Scala tem suporte nativo para classes parametrizadas com tipos. Essas classes genéricas são particularmente úteis para o desenvolvimento de classes collection.
+Aqui um exemplo que demonstra isso:
```

Comment:
Soa um pouco melhor (pelo menos pra mim) se mantermos o verbo to be. `Aqui temos um...`